### PR TITLE
English translation into Catalan and Spanish started, small problems have also been corrected

### PR DESCRIPTION
**Pull request by adriaroms**
Hello, English translation into Catalan and Spanish started, small problems have also been corrected.

> As a note: a tmp folder has been created with the files that have not yet been translated, if, when you want to translate a new file it is deleted from the tmp folder, when the whole project is translated, the tmp folder will be empty and will be you can delete it.

Adrià.

### CODE REVIEW

File: package-lock.json [hunk 1]
```diff
@@ -2203,6 +2203,11 @@
         "is-obj": "1.0.1"
```
Comment (johnleider) [inline package-lock.json]: please remove this file from the PR

**Comment by bad4iz** [comment]
should be divided into different PR. translation for `ca` and `es`. it would be better

File: package-lock.json (hunk 1 above)
Comment (adriaroms) [reply package-lock.json]: OK. I have taken the liberty of adding it to .gitignore, so it will not be added by mistake by any other contributor.

**Comment by adriaroms** [comment]
@bad4iz I could look to do it in different commits, but I think it is unnecessary, if @johnleider asks me to do it, there will be no problem and I will do it in future translations...

**Comment by johnleider** [comment]
Please split into 2 PRs

File: package-lock.json (hunk 1 above)
Comment (KaelWD) [reply package-lock.json]: He meant don't change it, not delete it.

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@ dist/
 public/
 npm-debug.log
 .idea/
+package-lock.json
```
Comment (johnleider) [inline .gitignore]: remove this, it's needed for our deployments

Comment (adriaroms) [reply .gitignore]: this weekend I'll finish the initial translations of style/ and resolve any conflicts that may arise

**Comment by johnleider** [comment]
Closing due to inactivity.

**Pull request closed without merging**